### PR TITLE
 fix(translate): #RBS-141 fix rbs i18n

### PR DESCRIPTION
**Pull request by AyaNADINI**
## Describe your changes
In order to ensure that our module can be delivered in both languages French and English,
We skim through the template folder and replace in html files what is hard-written (manually written) with i18n tags while implementing the i18n file

## Checklist tests
First, we create a resource and we make a reservation in Rbs.
We should have something like this when we click on our reservation: Réservation validée

## Issue ticket number and link
RBS-141 : https://entsupport.gdapublic.fr/browse/RBS-141

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/i18n/en.json [hunk 1]
```diff
@@ -439,7 +439,12 @@
     "rbs.calendar.sniplet.recurrence.multiday.edition.warning": "Resources will not be booked for the other occurrences of this event",
```
Comment (alicedraillard) [inline src/main/resources/i18n/en.json:439]: nom du commit : remplace fix(translate) par fix(i18n) plutôt

**Review by aymericdevriesere — APPROVED** [review]

File: src/main/resources/view-src/admin.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body ng-controller="RbsController" class="rbs">
 <admin-portal>
-    <h1>RBS</h1>
+    <h1>{{#i18n}}rbs.title.uppercase{{/i18n}}</h1>
```
Comment (BastienBenard) [inline src/main/resources/view-src/admin.html:12]: Est ce qu'il y a une raison de pourquoi on n'utilise pas la balise HTML ici ?

**Review by qpic-cgi — APPROVED** [review]

File: src/main/resources/view-src/admin.html (hunk 2 above)
Comment (AyaNADINI) [reply src/main/resources/view-src/admin.html:12]: c'est un cas spécial, la raison je sais pas vraiment

**Review by BastienBenard — APPROVED** [review]

File: src/main/resources/view-src/admin.html (hunk 2 above)
Comment (alicedraillard) [reply src/main/resources/view-src/admin.html:12]: C'est du mustache, utilisé pour l'envoi de notifications https://confluence.support-ent.fr/pages/viewpage.action?pageId=81474441

Comment (AyaNADINI) [reply src/main/resources/view-src/admin.html:12]: merci Alice

**Review by JinDouang — APPROVED** [review]

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**